<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>IIRFormInterface</class>
    <widget class="QWidget" name="IIRFormInterface">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>370</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
                <widget class="BodyLabel" name="BodyLabel_2">
                    <property name="text">
                        <string>通带</string>
                    </property>
                </widget>
            </item>
            <item row="0" column="1">
                <widget class="ComboBox" name="passbandsBox"/>
            </item>
            <item row="1" column="0">
                <widget class="BodyLabel" name="BodyLabel">
                    <property name="text">
                        <string>采样率</string>
                    </property>
                </widget>
            </item>
            <item row="1" column="1">
                <widget class="LineEdit" name="sampleRateEdit"/>
            </item>
            <item row="2" column="0">
                <widget class="BodyLabel" name="BodyLabel_3">
                    <property name="text">
                        <string>通带频率</string>
                    </property>
                </widget>
            </item>
            <item row="2" column="1">
                <widget class="LineEdit" name="passbandFreqLowEdit"/>
            </item>
            <item row="3" column="0">
                <widget class="BodyLabel" name="BodyLabel_5">
                    <property name="text">
                        <string>通带高频率</string>
                    </property>
                </widget>
            </item>
            <item row="3" column="1">
                <widget class="LineEdit" name="passbandFreqHighEdit"/>
            </item>
            <item row="4" column="1">
                <widget class="LineEdit" name="stopbandFreqLowEdit"/>
            </item>
            <item row="5" column="0">
                <widget class="BodyLabel" name="BodyLabel_6">
                    <property name="text">
                        <string>阻带高频率</string>
                    </property>
                </widget>
            </item>
            <item row="5" column="1">
                <widget class="LineEdit" name="stopbandFreqHighEdit"/>
            </item>
            <item row="6" column="0">
                <widget class="BodyLabel" name="BodyLabel_7">
                    <property name="text">
                        <string>通带波纹</string>
                    </property>
    </widget>
            </item>
            <item row="6" column="1">
                <widget class="LineEdit" name="passbandRippleEdit"/>
            </item>
            <item row="7" column="0">
                <widget class="BodyLabel" name="BodyLabel_8">
                    <property name="text">
                        <string>阻带衰减</string>
                    </property>
                </widget>
            </item>
            <item row="7" column="1">
                <widget class="LineEdit" name="stopbandAttenuationEdit"/>
            </item>
            <item row="8" column="0">
                <widget class="BodyLabel" name="BodyLabel_9">
                    <property name="text">
                        <string>原型</string>
                    </property>
                </widget>
            </item>
            <item row="8" column="1">
                <widget class="ComboBox" name="protoTypesBox"/>
            </item>
            <item row="4" column="0">
                <widget class="BodyLabel" name="BodyLabel_4">
                    <property name="text">
                        <string>阻带频率</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>ComboBox</class>
            <extends>QPushButton</extends>
            <header>qfluentwidgets</header>
        </customwidget>
        <customwidget>
            <class>BodyLabel</class>
            <extends>QLabel</extends>
            <header>qfluentwidgets</header>
        </customwidget>
        <customwidget>
            <class>LineEdit</class>
            <extends>QLineEdit</extends>
            <header>qfluentwidgets</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
